opal-ui/package-lock.json

```diff
@@ -1435,6 +1435,21 @@
       "integrity": "sha512-Tpp60P6IUJDTuOq/5Z8cdskzJujfwqfOTkrwIwj7IRISpnkJnT6SyJ4PCPnGMoFjC9ddhal5KVIYtAt97ix05A==",
       "dev": true
     },
+    "node_modules/body-parser/node_modules/qs": {
```

Comment:
Still in the package lock.

Reply:
This dep has always been there: `git diff 07003991723de735480f8cdb847750ac6c6d6e33 -- package-lock.json`